coding/file_container.cpp

```diff
@@ -169,7 +170,7 @@ MappedFile::Handle MappedFile::Map(uint64_t offset, uint64_t size, string const
 #else
   void * pMap = mmap(0, length, PROT_READ, MAP_SHARED, m_fd, alignedOffset);
   if (pMap == MAP_FAILED)
-    MYTHROW(Reader::OpenException, ("Can't map section:", tag, "with [offset, size]:", offset, size));
+    MYTHROW(Reader::OpenException, ("Can't map section:", tag, "with [offset, size]:", offset, size, "errno:", strerror(errno)));
```

Comment:
Sorry, maybe I missed something, but integral values of such things as ENOMEM, EINVAL, ENOFILE, etc. are not specified (there is only explicit specification for  symbolic values of some subset of possible system errors, http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/errno.h.html). For example, on my MacOS, EDQUOT has integral value 69, while on Linux it's value is 122. That's why I do not understand why do you insist on logging errno as integer.

By the way, could you please describe a situation when you need an integral value of errno?